### PR TITLE
Drop table command implementation

### PR DESCRIPTION
**Pull request by maheshrajamani**
**What this PR does**:
Implementation for drop table command

**Checklist**
- [x] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Comment by maheshrajamani** [comment]
Need to add validation to check if the provided table name is not a collection.

File: src/main/java/io/stargate/sgv2/jsonapi/service/operation/tables/DropTableOperation.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    implements Operation {
+  private static final Logger logger = LoggerFactory.getLogger(DropTableOperation.class);
+
+  private static final String DROP_TABLE_CQL = "DROP TABLE IF EXISTS \"%s\".\"%s\";";
+
+  @Override
+  public Uni<Supplier<CommandResult>> execute(
+      DataApiRequestInfo dataApiRequestInfo, QueryExecutor queryExecutor) {
+    logger.info("Executing DropTableOperation for {}", name);
+    String cql = DROP_TABLE_CQL.formatted(context.schemaObject().name.keyspace(), name);
```
Comment (Yuqi-Du) [inline src/main/java/io/stargate/sgv2/jsonapi/service/operation/tables/DropTableOperation.java:32]: variable 'name' here is confusing.
Recommend replace the [String name filed]  for DropTableOperation. 
Something like, tableName

**Review by Yuqi-Du — APPROVED** [review]
LGTM, one minor change may needed.

**Review by vkarpov15 — COMMENTED** [review]
LGTM :+1:

**Pull request merged**